dirty_cat/datasets/fetching.py

```diff
@@ -292,36 +299,36 @@ def _features_to_csv_format(features: Features) -> str:
 # Datasets fetchers section
 
 
-def fetch_employee_salaries():
+def fetch_employee_salaries() -> dict:
     """Fetches the employee_salaries dataset."""
     return fetch_openml_dataset(dataset_id=EMPLOYEE_SALARIES_ID)
 
 
-def fetch_road_safety():
+def fetch_road_safety() -> dict:
     """Fetches the road safety dataset."""
     return fetch_openml_dataset(dataset_id=ROAD_SAFETY_ID)
 
 
-def fetch_medical_charge():
+def fetch_medical_charge() -> dict:
     """Fetches the medical charge dataset."""
     return fetch_openml_dataset(dataset_id=MEDICAL_CHARGE_ID)
 
 
-def fetch_midwest_survey():
+def fetch_midwest_survey() -> dict:
     """Fetches the midwest survey dataset."""
     return fetch_openml_dataset(dataset_id=MIDWEST_SURVEY_ID)
 
 
-def fetch_open_payments():
+def fetch_open_payments() -> dict:
     """Fetches the open payments dataset."""
     return fetch_openml_dataset(dataset_id=OPEN_PAYMENTS_ID)
 
 
-def fetch_traffic_violations():
+def fetch_traffic_violations() -> dict:
     """Fetches the traffic violations dataset."""
     return fetch_openml_dataset(dataset_id=TRAFFIC_VIOLATIONS_ID)
 
 
-def fetch_drug_directory():
+def fetch_drug_directory() -> dict:
```

Comment:
Small comment: refrain from doing changes not related to the purpose of a PR. They make the PR harder to review.

You can make those changes in another PR (these changes are indeed useful).